Makefile

```diff
@@ -16,7 +16,7 @@ clean:
 # Builds
 
 build: setup-git
-	@cargo +stable build --all-features
+	cargo +stable build --all-features
```

Comment:
This change is unrelated, can we revert?

Reply:
I found it useful to see what we're actually building, it's the same behavior we have for `test` and `lint` targets.